.github/workflows/run-tests-on-windows.yml

```diff
@@ -251,11 +251,6 @@ jobs:
         continue-on-error: true
         if: always()
         run: |
-          tasklist /FI "IMAGENAME eq python3.exe" /NH | findstr /I python3.exe >nul
-          if %errorlevel% equ 0 (
-              wmic process where "name='python3.exe'" delete
-              echo Python3 process has been terminated.
-          ) else (
-              echo No Python3 process was found.
-          )           
+          wmic process where "name='python.exe' or name='python3.exe'" delete >nul 2>&1
+          if %errorlevel%==0 (echo Python processes terminated) else (echo No Python processes found)
```

Comment:
```suggestion
          taskkill /F /IM "python.exe" /IM "python3.exe" >nul 2>&1
          if %ERRORLEVEL% EQU 0 (echo Python processes terminated) else (echo No Python processes found)
          exit /b 0
```

Reply:
`wmic` is deprecated, and is currently being removed from Windows, should use `taskkill` instead.